app/controllers/api/v1/organization_pages_controller.rb

```diff
@@ -1,33 +1,17 @@
 class Api::V1::OrganizationPagesController < Api::ApiController
-  require_authentication :update, :create, :destroy, scopes: [:organization]
-
-  resource_actions :default
-
-  schema_type :strong_params
-
-  allowed_params :create, :url_key, :title, :content, :language
-  allowed_params :update, :url_key, :title, :content, :language
+  include Pages
 
-  before_filter :set_language_from_header, only: [:index]
-
-  def controlled_resources
-    @controlled_resouces ||= super.where(organization: params[:organization_id])
-  end
-
-  protected
+  require_authentication :update, :create, :destroy, scopes: [:organization]
 
-  def build_resource_for_create(create_params)
-    create_params[:links] ||= {}
-    create_params[:links][:organization] = params[:organization_id]
-    super create_params
+  def parent_resource
+    :organization
   end
 
-  def link_header(resource)
-    resource = resource.first
-    send(:"api_#{ resource_name }_url", id: resource.id, organization_id: resource.organization_id)
+  def resource_id
+    params[:organization_id]
   end
 
-  def set_language_from_header
-    params[:language] = "en"
+  def resource_name
+    "organization_page"
```

Comment:
Should we use a class constant for this instead? It's been a while with mem improvements but don't string allocations add up over time?